### ISSUE TITLE
Update 2.0.0 to three.js 0.170.0

### ISSUE THREAD

**Issue by crystalfp** [Fennec-hub/three-viewport-gizmo#17]
Installing three-viewport-gizmo gives the following warning (after adding --force option):
```
> npm up --force
npm warn using --force Recommended protections disabled.
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: three-viewport-gizmo@2.0.0
npm warn Found: three@0.170.0
npm warn node_modules/three
npm warn   three@"^0.170.0" from the root project
npm warn   2 more (camera-controls, troika-three-text)
npm warn
npm warn Could not resolve dependency:
npm warn peer three@"^0.162.0" from three-viewport-gizmo@2.0.0
npm warn node_modules/three-viewport-gizmo
npm warn   three-viewport-gizmo@"^2.0.0" from the root project
npm warn
npm warn Conflicting peer dependency: three@0.162.0
npm warn node_modules/three
npm warn   peer three@"^0.162.0" from three-viewport-gizmo@2.0.0
npm warn   node_modules/three-viewport-gizmo
npm warn     three-viewport-gizmo@"^2.0.0" from the root project
npm warn ERESOLVE overriding peer dependency
```

**Comment by Fennec-hub** [comment]
The problem seems to be related with how three.js versioning works, when to major version is 0, semver treats the second digit (0.x.0) as the breaking change marker, so even if I update to 0.170.0 the problem will remain, the only solution I can think of at the moment is use this alternative `>=0.162.0 <1.0.0`

**Comment by Fennec-hub** [comment]
Fixed and tested with `v2.0.2`, 

Many thanks for the heads up!
